package.json

```diff
@@ -75,6 +75,7 @@
   "resolutions": {
     "carbon-addons-iot-react/chokidar": "3.3.1",
     "carbon-addons-iot-react/react-grid-layout": "1.2.2",
+    "carbon-addons-iot-react/flatpickr": "4.6.1",
```

Comment:
just FYI, specifying resolution this way didn't do anything. The install never picked up as expected. That's why I did it for all sub-packages a few line below this. see line 82. 4.6.1 is also what angular is using currently.

Reply:
Hmm. We should test this to confirm. Health app was seeing this issue before this change and have confirmed that this fix solved the issue.

Reply:
My suspicion is that you adding it to the `react` package.json resolutions is the fix. I checked the yarn.lock file multiple times when I added that pin in multiple formats and the package name in the `root` package resolution didn't change it. It might just be because it wasn't being consumed though. IDK.